### PR TITLE
added admin page

### LINKED ISSUE #89: Admin Page
Please refer to the design. Make sure the UI is pixel perfect. Follow BEM notation.

### PR DESCRIPTION
**Pull request by sonali-singh97**
### Fixes #89 

**Description** <br/>
Added Admin page.

**Screenshots** <br/>

screencapture-localhost-3000-admin-2020-10-03-03_30_16

### CODE REVIEW

**Review by salil-naik — CHANGES_REQUESTED** [review]
- [ ] This is about page so rename **admin** to **about.** 

- [ ] Learn about foundation css classes. Especially 'large-', 'medium-', 'small-'. You seem to be confused. Understand where you need to use which classes. Also learn when to use large, medium and small. 

- [ ] Format using [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)

- [ ] You have used `p` instead of `ul` and `li` for listing the achievements of the cofounders. Fix that.

- [ ] Remove spaces from the image names. Rename the images wherever required.  `image 1.png` is wrong. `image-1.png` or `image_1.png` is the right practice.

**Comment by sonali-singh97** [comment]
Okay, I will make these changes.

File: frontend/src/pages/AboutUs.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                            </div>
+                        </div>
+                    </div>
+
+
+                </div>
+
+                <div style={{ background: "linear-gradient(180deg, #FFFDF6 0%, #FFEBF3 100%)" }}>
+                    <div className="coFounder">
+                        <div className="grid-container grid-x grid-margin-x ">
```
Comment (salil-naik) [inline frontend/src/pages/AboutUs.jsx:43]: grid-container and grid-x cannot be together. 

```jsx
<div className="grid-container">
    <div className="grid-x grid-margin-x">
     // cell will come here. 
    </div>
</div>
```

File: frontend/src/pages/AboutUs.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                            <div className="cell ">
+                                <h1 >
+                                    Co-Founders
+                            </h1>
+                                <br />
+                            </div>
+
+                            <div className="cell medium-6" style={{ paddingRight: "60px" }}>
```
Comment (salil-naik) [inline frontend/src/pages/AboutUs.jsx:51]: never add custom styles to foundation classes. Remove paddingRight. If the code breaks, it's fine. I'll fix it.

**Review by salil-naik — CHANGES_REQUESTED** [review]
Please do the changes.

File: frontend/src/pages/AboutUs.jsx (hunk 1 above)
Comment (sonali-singh97) [reply frontend/src/pages/AboutUs.jsx:43]: Alright, I will do the changes

File: frontend/src/pages/AboutUs.jsx (hunk 2 above)
Comment (sonali-singh97) [reply frontend/src/pages/AboutUs.jsx]: I have changed it kindly review it

**Review by salil-naik — APPROVED** [review]

**Pull request merged**